lib/Mojo/Util.pm

```diff
@@ -37,11 +38,23 @@ use constant {
 # To generate a new HTML entity table run this command
 # perl examples/entities.pl
 my %ENTITIES;
-for my $line (split "\n", join('', <DATA>)) {
-  next unless $line =~ /^(\S+)\s+U\+(\S+)(?:\s+U\+(\S+))?/;
-  $ENTITIES{$1} = defined $3 ? (chr(hex $2) . chr(hex $3)) : chr(hex $2);
+{
+  # Don't use Mojo::File here due to circular dependencies
+  my $html_path
+    = File::Spec->catfile(File::Basename::dirname(Cwd::realpath(__FILE__)),
```

Comment:
Why `Cwd::realpath` here?

Reply:
It handles the edge case where Mojo::Util is loaded via a symlink to still reference the correct location

Reply:
How could that ever happen? Until your recent addition of `curfile` we did not use `Cwd::realpath` for any of this and there were never any issues.

Reply:
It does seem to work the same if I remove `Cwd::realpath`.